src/ToastNotification/ToastNotificationContainer.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace PackagedUi\Fusion\ToastNotification;
+
+use Packaged\Glimpse\Tags\Div;
+use Packaged\Ui\Html\HtmlElement;
+use PackagedUi\BemComponent\BemComponentTrait;
+use PackagedUi\Fusion\Component;
+use PackagedUi\Fusion\ComponentTrait;
+
+class ToastNotificationContainer extends Div implements Component
+{
+  use BemComponentTrait;
+  use ComponentTrait;
+
+  /** @var ToastNotification[] */
+  protected $_toastNotifications = [];
+
+  /** @var int */
+  protected $_delay = 5000;
```

Comment:
why are we defaulting to 5s?

Reply:
It defaults to 5s on TAV which I'am assuming they will want to keep. Can make it 0 default I suppose but the use case for it down the line would require the delay to be added to all instances